samples/bluetooth/peripheral_hids_mouse/bt_ll_softdevice_hci_rpmsg.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Configuration file passed as an overlay when building
+# zephyr/samples/bluetooth/hci_rpmsg as a child image.
+
+CONFIG_BT_MAX_CONN=4
```

Comment:
The sample is only Bluetooth peripheral, why do we need 4 conn if slave count equals 2?

Reply:
The same for keyboard

Reply:
If you want to use two peripheral connection you still need have one slot for central connection so minimal count of connection is 3 here but for symmetry I set it to 4